js/src/components/paid-ads/validateCampaign.js

```diff
@@ -23,6 +27,26 @@ const validateCampaign = ( values ) => {
 		);
 	}
 
+	if (
+		Number.isFinite( values?.amount ) &&
+		Number.isFinite( opts?.dailyBudget )
+	) {
+		const { amount } = values;
+		const { dailyBudget } = opts;
+		const minAmount = Math.round( dailyBudget * BUDGET_MIN_PERCENT, 2 );
+
+		if ( amount < minAmount ) {
+			errors.amount = sprintf(
+				/* translators: %s: minimum daily budget */
+				__(
+					'Please make sure daily average cost is greater than %s.',
```

Comment:
I think we should add the currency here, so it reads:

"Please make sure daily average cost is greater than {minAmount} {currency}."

Reply:
Thanks @joemcgill . I've pushed an update here:
https://github.com/woocommerce/google-listings-and-ads/pull/2583/commits/603d9c4c192278c0bc7d0e69cc4ecab0025d4a60
Feel free to let me know if it looks good to you.